src/System.IO.FileSystem/tests/Path/GetPathRoot.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Path_GetPathRoot : FileSystemTest
+    {
+        [Fact]
+        public void NullAsPath_ReturnsNull()
+        {
+            Assert.Null(Path.GetPathRoot(null));
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(" ")]
+        public void IllegalPath_ThrowsArgumentException(string path)
+        {
```

Comment:
I'm already adding tests on: https://github.com/dotnet/corefx/pull/19343

We could keep this one, or the other, whichever you prefer.